Cython/Compiler/CmdLine.py

```diff
@@ -218,7 +224,17 @@ def get_param(option):
         bad_usage()
     if Options.embed and len(sources) > 1:
         sys.stderr.write(
-            "cython: Only one source file allowed when using -embed\n")
+            "cython: Only one source file allowed when using --embed\n")
         sys.exit(1)
+    if options.module_name:
+        if options.timestamps:
+            sys.stderr.write(
+                "cython: Cannot use --module-name with --timestamps\n")
+            sys.exit(1)
+        if len(sources) > 1:
+            sys.stderr.write(
+                "cython: Only one source file allowed when using --module-name\n")
+            sys.exit(1)
+            parser.error("")
```

Comment:
?

Reply:
It's a backport artefact apparently, sorry for the oversight.
```suggestion
            sys.exit(1)
```